index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method, or to use a
+different mechanism to validate whether the <a>holder</a> is bound to the
+presented <a>claims</a>, if required. Such decision can prevent impact on the
+<a>verifier</a>'s liability or lack thereof if not specified by other means
+such as a <code>termsOfUse</code> policy.
+        </p>
+
+        <p>
+This specification defines the <code>confirmationMethod</code> <a>property</a>
+for expressing confirmation method information in a
+<code>credentialSubject</code> in a <a>verifiable credential</a>.
+        </p>
+
+        <div class="note">
+          <p>
+For example, an <a>issuer</a> can include a confirmation method based on public
+key cryptography in the <a>verifiable credential</a>. A <a>holder</a> can
+generate and include a <a>proof</a> with a cryptographic signature in the
+<a>verifiable presentation</a>, where the verification key of the cryptographic
+signature is bound to a confirmation method in the embedded <a>verifiable
+credential</a>.
+          </p>
+          <p>
+A <a>verifier</a> can validate that the <a>holder</a> controls
+or has been designated the ability to use the confirmation method
+by verifying the <a>proof</a> of the <a>verifiable
+presentation</a> using the information in the confirmation method. The
+confirmation method can include the verification key, or the type of the
+confirmation method can define that the verification key is to be inferred from
+other <a>properties</a> in the <a>verifiable credential</a>, such as the
+<code>credentialSubject.id</code>.
+          </p>
+        </div>
+
+        <dl>
+          <dt><dfn>confirmationMethod</dfn></dt>
+          <dd>
+          <dd>
+            <p>
+If present, the value of the <code>confirmationMethod</code> <a>property</a> is
+one or more confirmation methods, each providing enough information for a
+<a>verifier</a> to validate that the <a>holder</a> generating a <a>verifiable
+presentation</a> has proven control of a confirmation method bound to one or
+more <a>claims</a> in a <a>verifiable credential</a> in the
+<a>verifiable presentation</a>. It is required that the <a>issuer</a> verifies
+the <a>holder</a> can satisfy each <code>confirmationMethod</code> they include
+in the <a>claims</a> of the <a>verifiable credential</a>s they issue.
+            </p>
+            <p>
+Each confirmation method MUST specify its <code>type</code> (for example,
+<code>DIDAuthWithSubjectIdConfirmation</code>) and MAY specify an
+<code>id</code>. The precise <a>properties</a> and semantics of each
+confirmation method are determined by the specific
+<code>confirmationMethod</code> type definition.
+            </p>
+          </dd>
+        </dl>
+
+        <p>
+The type <code>DIDAuthWithSubjectIdConfirmation</code> defines that a
```

Comment:
Why do we need a separate type for DIDAuth? I thought in F2F, the discussion was such that we add a text that when a key material is used as a subject identifier, it can also be used for holder binding?

Reply:
Without any guidance, I believe we cannot add language like that. A verifier simply doesn't know what the `id` in the `credentialSubject` represents in that case. That is the reason why a feature like `confirmationMethod` is needed.

Reply:
I guess this is addressed (see below)